src/containers/remixes-page/RemixesPageProvider.tsx

```diff
@@ -96,7 +94,7 @@ const RemixesPageProvider = ({
       actions: tracksActions,
       scrollParent: containerRef as any,
       loadMore: (offset: number, limit: number) => {
-        loadMore(offset, limit, { trackId })
+        loadMore(offset, limit, { trackId: originalTrack?.track_id || null })
```

Comment:
i think this is probably the same behavior as before, but definitely a little odd.

we do have this withNullGuard construct that we've used in a full places to prevent from getting into this annoying situation https://github.com/AudiusProject/audius-client-old/blob/ad02f295906122c234ae5a0bd593c684129dce17/src/containers/remixes-page/components/desktop/RemixesPage.tsx#L33 (used here but a level down in the mobile/desktop rather than at the provider level)

Reply:
also nit, if we keep this

originalTrack?.track_id ?? null probs better